optimum/exporters/openvino/model_configs.py

```diff
@@ -2963,6 +2963,21 @@ class GLMOpenVINOConfig(LlamaOpenVINOConfig):
     MIN_TRANSFORMERS_VERSION = "4.46.0"
 
 
+@register_in_tasks_manager(
+    "glm4",
+    *[
+        "feature-extraction",
+        "feature-extraction-with-past",
+        "text-generation",
+        "text-generation-with-past",
+        "text-classification",
+    ],
+    library_name="transformers",
+)
+class GLMOpenVINOConfig(LlamaOpenVINOConfig):
+    MIN_TRANSFORMERS_VERSION = "4.52.0"
```

Comment:
why not :
```suggestion
    MIN_TRANSFORMERS_VERSION = "4.51.3"
```
https://github.com/huggingface/transformers/blob/v4.51.3/src/transformers/models/glm4/modeling_glm4.py

also optimum-intel is compatible with v4.51 since https://github.com/huggingface/optimum-intel/pull/1242